tests/builder/help.rs

```diff
@@ -351,6 +354,152 @@ Options:
     utils::assert_output(cmd, "ctest --help", DEFAULT_HELP, false);
 }
 
+#[test]
+fn try_help_default() {
```

Comment:
These might need
```rust
#[cfg(feature = "error-context")]
```

Reply:
Would it fail in CI if it needed to be added?

Reply:
Huh, something broke and we aren't testing that case anymore.  Oh well.